Jenkinsfile

```diff
@@ -61,10 +61,12 @@ pipeline {
 
     stage('Check') {
       steps {
-        withCheckEnvironment {
-          gradle 'check'
+        catchError(buildResult: 'UNSTABLE', stageResult: 'FAILURE') {
```

Comment:
Why do we set the build to unstable, but the stage to failure? Is the rest of the stage still executed, after setting a stage result?